escrow/state_service_test.go

```diff
@@ -82,6 +101,18 @@ func TestGetChannelState(t *testing.T) {
 
 	assert.Nil(t, err)
 	assert.Equal(t, stateServiceTest.defaultReply, reply)
+	//Channel's nonce  =  blockchain nonce + 1
+	stateServiceTest.defaultChannelData.Nonce = big.NewInt(4)
+	reply, err = stateServiceTest.service.GetChannelState(
+		nil,
+		stateServiceTest.defaultRequest,
+	)
+	assert.Nil(t, err)
+	assert.NotNil(t, reply)
```

Comment:
I would check that reply contains expected data not just it is not `nil`

Reply:
Makes , Sense done